pkg/sbom/sbom.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.31.0
-// 	protoc        v4.24.2
+// 	protoc        v4.24.0
```

Comment:
the presubmit is failing because the proto is getting regenerated with a different (older) protoc:

```
-// 	protoc        v4.24.0
+// 	protoc        v4.24.2
```